policyengine_us/parameters/gov/states/ms/tax/income/exemptions/regular/amount.yaml

```diff
@@ -0,0 +1,19 @@
+description: MS provides a regular exemption of $6k for individuals, $12k for couples, $8k for heads of household.
+metadata:
+  unit: currency-USD
+  label: MS Regular Exemption
+  reference: 
+    - title: Mississippi State FORMS 80-105 AND 80-205
```

Comment:
Can you find a legal reference?

Reply:
Sure!

Reply:
I mean from https://advance.lexis.com/container?config=00JAAzNzhjOTYxNC0wZjRkLTQzNzAtYjJlYS1jNjExZWYxZGFhMGYKAFBvZENhdGFsb2cMlW40w5iIH7toHnTBIEP0&crid=2fcdae0c-c595-411f-aa8c-6818383d928e